test/Webhook.spec.js

```diff
@@ -48,6 +48,21 @@ describe('Webhooks', () => {
         expect(event.id).to.equal(EVENT_PAYLOAD.id);
       });
 
+      it('should return an Event instance from a payload and header with type Buffer', async () => {
+        const header = stripe.webhooks.generateTestHeaderString({
```

Comment:
let's also add tests that create and pass instances of Uint8Array

Reply:
Do we have file upload tests? Can we also add Uint8Array variants there?

Reply:
Added a test for passing in file data as `Uint8Array` in flows.spec.js